assets/app/scripts/filters/util.js

```diff
@@ -117,6 +117,9 @@ angular.module('openshiftConsole')
           }
           unit += (amount === "1" ? "core" : "cores");
           break;
+        case "storage":
+          unit = unit.substring(0, 1);
```

Comment:
i'd expect this to behave like memory and add the "B" to it so you see GB instead of G, what units are actually being returned, since I see you are substring-ing it